gateway/src/apicast/policy/upstream/upstream.lua

```diff
@@ -22,7 +22,7 @@ local function init_config(config)
     local upstream, err = Upstream.new(rule.url)
 
     if upstream then
-      tab_insert(res, { regex = rule.regex, upstream = upstream })
+      tab_insert(res, { regex = rule.regex, upstream = upstream, url = rule.url })
```

Comment:
I think that because of the change below we no longer need to store `upstream` here.

Reply:
Cleaned up.